assets/queries/ansible/aws/db_security_group_public_scope/test/negative.yaml

```diff
@@ -0,0 +1,21 @@
+- name: example ec2 group2
+  ec2_group1:
+    name: example1
+    description: an example EC2 group
+    vpc_id: 12345
+    region: eu-west-1a
+    aws_secret_key: SECRET
+    aws_access_key: ACCESS
+    rules:
+      - proto: tcp
+        from_port: 80
+        to_port: 80
+        cidr_ip: 10.1.1.1/32
+  rules_egress:
+      - proto: tcp
+        from_port: 80
+        to_port: 80
+        cidr_ip: 10.1.1.1/32
+        group_name: example-other
+        # description to use if example-other needs to be created
+        group_desc: other example EC2 group
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline